libcloud/compute/types.py

```diff
@@ -204,56 +214,42 @@ class NodeState(object):
     :cvar PAUSED: Node is paused.
     :cvar UNKNOWN: Node state is unknown.
     """
-    RUNNING = 0
-    REBOOTING = 1
-    TERMINATED = 2
-    PENDING = 3
-    UNKNOWN = 4
-    STOPPED = 5
-    SUSPENDED = 6
-    ERROR = 7
-    PAUSED = 8
-
-    @classmethod
-    def tostring(cls, value):
-        values = cls.__dict__
-        values = dict([(key, string) for key, string in values.items() if
-                       not key.startswith('__')])
-
-        for item_key, item_value in values.items():
-            if value == item_value:
-                return item_key
-
-    @classmethod
-    def fromstring(cls, value):
-        return getattr(cls, value.upper(), None)
-
-
-class StorageVolumeState(object):
+    RUNNING = 'running'
```

Comment:
Are we ok with this behavior change?  The ```NodeState.tostring(node.state)``` values previously were all-caps, now they're all lower-case.  I know this means my team's codebase will require changes to keep existing behavior.  Could we  make ```Type.tostring``` return ```value.upper()```?